docker/Dockerfile-arm64v8

```diff
@@ -0,0 +1,23 @@
+FROM arm64v8/openjdk:11
```

Comment:
We're switching our Docker image use Zulu: https://github.com/prestosql/presto/pull/4642/files

Also, we use CentOS for x86, so it seems better to be consistent across architectures for both OS and JVM.

Additionally, I'm wary of the `openjdk` images since in the past they have used pre-release versions of the JVM but the version reported itself as a stable release: https://mail.openjdk.java.net/pipermail/jdk8u-dev/2019-May/009330.html

I'm not sure if the situation for the `opendk` Docker images has changed, but I have far more trust in the Zulu builds that come from Azul.